packages/events/defaultSubscribers/dashboard.js

```diff
@@ -20,35 +13,25 @@ module.exports = {
           // TODO: Do we care about ID collisions?
           this.pendingTransactions[payload.id] = payload;
 
-          this.spinner = this.getSpinner({
+          spinners.add("dashboard-handler", {
             text: `Waiting for transaction signature. Please check your wallet for a transaction approval message.`,
-            color: "red",
-            stream: new Writable({
-              write: function (chunk, encoding, next) {
-                this._logger.log(chunk.toString());
-                next();
-              }.bind(this)
-            })
+            color: "red"
           });
         }
       }
     ],
     "rpc:result": [
       function (event) {
-        const { payload, error, result } = event;
+        let { payload, error, result } = event;
```

Comment:
Do all variables have to be mutable?

Reply:
No - could break it out, but I like the readability of the destructuring syntax.